server/index.js

```diff
@@ -105,9 +113,17 @@ app.delete('/api/:userId/:hotelId', (req, res, next) => {
       `;
   return db.query(sql)
     .then(result => {
-      res.status(201).json(result.rows);
+      res.json(result.rows)
+      res.status(204)
```

Comment:
Are you sure that the proper status code is being used? Did you test this with HTTPie?